app/models/sign_in/client_config.rb

```diff
@@ -16,6 +16,7 @@ class ClientConfig < ApplicationRecord
     validates :authentication,
               presence: true,
               inclusion: { in: Constants::Auth::AUTHENTICATION_TYPES, allow_nil: false }
+    validates :shared_sessions, inclusion: [true, false]
```

Comment:
I think this is fine. `presence` won't work because it will reject `false` and this `inclusion` will catch `nil` but allow `false`. I agree that `nil` _should_ probably never happen especially with a default value but who knows 😱

Reply:
yeah this is fine (but of course I'd say that because this is a pattern I follow, ha). The reason to do this is just that I prefer having errors caught at the app level before getting to the database level, even if the database would definitely catch this as well